README.md

```diff
@@ -100,31 +173,45 @@ Drop the content of `name-1.txt`, verify it is gone, recreate it via
 > cat name-1.txt
 ``` 
 
-The command `datalad make` does also support to just record the parameters
-that would lead to a certain computation, without actually performing the
-computation. We refer to this as *speculative computation*.
-
-To use this feature, the following configuration value has to be set:
+The `datalad make` command can also be used to perform a *speculative
+computation*. To use this feature, the following configuration value 
+has to be set:
 
 ```bash
 > git config annex.security.allow-unverified-downloads ACKTHPPT
 ```
 
-Afterward, a speculative computation can be recorded by providing the `-u` option
-(url-only) to `datalad make`.
+Afterwards, a speculative computation can be initiated by using the `-u`
+option:
 
 ```bash
 > datalad make -p first=john -p second=susan -p output=person \
 -o person-1.txt -o person-2.txt -u one-to-many
-> cat person-1.txt    # this will fail, because the computation has not yet been performed
 ```
 
-`ls -l person-1.txt` will show a link to a not-downloaded URL-KEY.
-`git annex whereis person-1.txt` will show the associated computation description URL.
-No computation has been performed yet, `datalad make` just creates an URL-KEY and
-associates a computation description URL with the URL-KEY.
+This will fail, because no computation has been performed, and the file content
+is unavailable:
+
+```bash
+> cat person-1.txt
+```
+
+However, `ls -l` will show a symlink to a URL-KEY:
+
+```bash
+> ls -l person-1.txt
+```
```

Comment:
Either in addition or instead of `ls -l`, we could also show `git annex info`. Here's what it does with a different kind of an URL key (`URL--yt`):

```
❱ git annex info 2024-04-04_00_livestream.mp4
file: 2024-04-04_00_livestream.mp4
size: 0 bytes (+ 1 unknown size)
key: URL--yt:https://www.youtube.com/live/BwRy3z_hQ70,63si,61IueRNywRKGSzBc7U
present: false
```

Reply:
This is done in af9073a.